Source/Utils/WarpXProfilerWrapper.H

```diff
@@ -11,7 +11,8 @@
 #include "AMReX_BLProfiler.H"
 #include "AMReX_GpuDevice.H"
 
-static void doDeviceSynchronize(int do_device_synchronize)
+AMREX_FORCE_INLINE
+void doDeviceSynchronize(int do_device_synchronize)
```

Comment:
Fixes the warning:
`./Source/Utils/WarpXProfilerWrapper.H:14:13`: warning: ‘void doDeviceSynchronize(int)’ defined but not used [-Wunused-function]
 static void doDeviceSynchronize(int do_device_synchronize)